templates/flutter/lib/client.dart.twig

```diff
@@ -134,7 +134,9 @@ class Client {
             await this.init();
         }
 
-        params.removeWhere((key,value) => value == null);
+        if(params.isNotEmpty()) {
```

Comment:
In dart this is a getter and not a function ☺️ 

```suggestion
        if(params.isNotEmpty) {
```

Reply:
Thanks